dipy/workflows/reconst.py

```diff
@@ -295,8 +343,23 @@ def run(self, input_files, bvalues, bvectors, mask_files,
             multiple masks at once. (default: No mask used)
         b0_threshold : float, optional
             Threshold used to find b=0 directions
+        bvecs_tol : float, optional
+            Bvecs should be unit vectors. (default:0.01)
+        roi_center : variable int, optional
+            Center of ROI in data. If center is None, it is assumed that it is
+            the center of the volume with shape `data.shape[:3]` (default None)
+        roi_radius : int, optional
+            radius of cubic ROI in voxels (default 10)
+        fa_thr : float, optional
+            FA threshold for calculating the response function (default 0.7)
         frf : tuple, optional
-            Fiber response function to me mutiplied by 10**-4 (default: 15,4,4)
+            Fiber response function 15, 4, 4 to be mutiplied by 10**-4 
+            (default: None)
+        sh_order : int, optional
+            Spherical harmonics order (default 6) used in the CSA fit.
+        odf_to_sh_order : int, optional
```

Comment:
I don't know if that's just my setup, but the help message for this appears for the `--out_dir` argument, and the `out_dir` help message appears for this argument...

Reply:
There was no problem with setup. When you create the docstrings of workflows they have to be in the same order as the input parameters. Should be corrected now.